transmogrifier/config.py

```diff
@@ -83,10 +85,30 @@ def get_transformer(source: str) -> type:
     return getattr(source_module, class_name)
 
 
-def load_external_config(path: str) -> dict:
+def load_external_config(file_path: str, file_type: str) -> Union[dict, Tag]:
     """
-    Return dict from JSON config file.
+    Return dict from a config file.
 
     """
-    with open(path, "rb") as json_file:
-        return json.load(json_file)
+    if file_type == "json":
+        with open(file_path, "rb") as json_file:
+            return json.load(json_file)
+    elif file_type == "xml":
+        with open(file_path, "rb") as xml_file:
+            return BeautifulSoup(xml_file, "xml")
+    else:
+        return {}
+
+
+def create_dict_from_xml_config(
+    xml_content: Tag, element_name: str, key_tag: str, value_tag: str
+) -> dict:
+    """
+    Return dict from XML object that is formatted like the country and language code
+    XML files hosted by Library of Congress.
+
+    """
+    config_dict = {}
+    for element in xml_content.find_all(element_name):
+        config_dict[element.find(key_tag).string] = element.find(value_tag).string
```

Comment:
There's a ticket to go through the entire codebase and do this correctly, but in this case it's important now to go ahead and convert these bs4 `NavigableString` objects to Unicode strings because these config dicts will be stored in memory throughout the entire transform process. [Per the bs4 documentation](https://www.crummy.com/software/BeautifulSoup/bs4/doc/#navigablestring), each `NavigableString` object "will carry around a reference to the entire Beautiful Soup parse tree, even when you’re done using Beautiful Soup. This is a big waste of memory."